tests/assets/spa/dt/fetch-dt-crossorigin-load-2.html

```diff
@@ -7,8 +7,8 @@
   <head>
     <meta charset="utf-8">
     <title>Example Page</title>
-    {config}
     {init}
+    {config}
```

Comment:
Does this end up affecting the configurations in any way? I removed the sslShim in my branch because it's inserted twice ultimately.